lib/relation-definition.js

```diff
@@ -1991,15 +1994,15 @@ HasOne.prototype.related = function(condOrRefresh, options, cb) {
 
 /**
  * Define a Promise-based method for the hasOne relation itself
- * - order.customer.getAsync(cb): Load the target model instance asynchronously
+ * - order.customer.get(cb): Load the target model instance asynchronously
  *
  * @param {Function} cb Callback of the form function (err, inst)
  * @returns {Promise | Undefined} Returns promise if cb is omitted
  */
-
-HasOne.prototype.getAsync = function(options, cb) {
+HasOne.prototype.getAsync = // deprecated alias
```

Comment:
I think this alias is not needed because `HasOne` class is not part of the public API.